wdio-cucumber/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "SauceLabsVisualExampleforWebdriverIOCucumber",
```

This is the correct format (even if others don't respect it).
```suggestion
  "name": "saucelabs-visual-example-webdriverio-cucumber",
```